dotCMS/src/main/webapp/html/portlet/ext/workflows/view_workflow_task.jsp

```diff
@@ -42,7 +41,16 @@
 	List<WorkflowComment> comments = APILocator.getWorkflowAPI().findWorkFlowComments(task);
 	List<WorkflowHistory> history = APILocator.getWorkflowAPI().findWorkflowHistory(task);
 	Collections.reverse(history);
-	List<IFileAsset> files = APILocator.getWorkflowAPI().findWorkflowTaskFilesAsContent(task, user); // new files
+	List<IFileAsset> files = new ArrayList<>();

```

Comment:
why you give memory to the List if it is gonna be override in the next line, if you need an empty list in case of error you can use Collections.EMPTY_LIST on the catch